tests/NuGet.Services.AzureSearch.Tests/SearchDocumentBuilderFacts.cs

```diff
@@ -138,6 +138,42 @@ public async Task SetsExpectedProperties()
             }
         }
 
+        public class UpdateOwners : BaseFacts
+        {
+            public UpdateOwners(ITestOutputHelper output) : base(output)
+            {
+            }
+
+            [Fact]
+            public async Task SetsExpectedProperties()
+            {
+                var document = _target.UpdateOwners(
```

Comment:
may be also assert the expected owners list before invoking the `UpdateOwners` that way we know its changed?

Reply:
The rest of the tests assume they are constant. Will leave as-is.